lapis2-docs/src/components/RenderedExampleForm.astro

```diff
@@ -0,0 +1,20 @@
+---
+import { getConfig } from '../config';
+import { getLapisUrl } from '../lapisUrl';
+
+const firstMetadatum = getConfig().schema.metadata[0];
+const lapisUrl = getLapisUrl();
+
```

Comment:
Can we make here a note/heading, where it says "this is the code" (but with better words). And then have a note/heading below, that says, "this is the expected result"? It feels a bit off floating there without any description.